src/ws_correction.py

```diff
@@ -229,9 +229,15 @@ def move_wrong_scheme_messages(user, data, coda_input_dir):
             # For each rqa message, create a copy of this td, append the rqa message, and add this to the
             # list of TracedData.
             for target_field, update in rqa_updates:
+                for plan in PipelineConfiguration.RQA_CODING_PLANS:
+                    if plan.raw_field == update.source:
+                        break
+                else:
```

Comment:
Weird indentation, I'm surprised that this parses

Reply:
I think that comment tells me everything I need to know about the usability of that construct. I'll re-write more sanely.

Reply:
I changed to this:
```python
raw_field_to_rqa_plan = {plan.raw_field: plan for plan in PipelineConfiguration.RQA_CODING_PLANS}
for target_field, update in rqa_updates:
    plan = raw_field_to_rqa_plan[update.source]
```
which I think is much cleaner